huaweicloud/provider.go

```diff
@@ -473,7 +473,7 @@ func Provider() *schema.Provider {
 			"huaweicloud_compute_keypair":                  ResourceComputeKeypairV2(),
 			"huaweicloud_compute_servergroup":              ResourceComputeServerGroupV2(),
 			"huaweicloud_compute_eip_associate":            ResourceComputeFloatingIPAssociateV2(),
-			"huaweicloud_compute_volume_attach":            ResourceComputeVolumeAttachV2(),
+			"huaweicloud_compute_volume_attach":            ResourceComputeVolumeAttach(),
```

Comment:
suggest to keep the name to `ResourceComputeVolumeAttachV2`, this change will break other clouds.

Reply:
The v2 version interface is not used under this resource now, so the v2 suffix is inaccurate.